pkgs/top-level/all-packages.nix

```diff
@@ -7872,11 +7872,13 @@ with pkgs;
 
   avrgcclibc = throw "avrgcclibs are now separate packages, install avrbinutils, avrgcc and avrlibc";
 
-  avrbinutils = callPackage ../development/misc/avr/binutils {};
+  avrbinutils      = callPackage ../development/misc/avr/binutils {};
```

Comment:
Whenever multiple defs are aligned with extra spaces before the `=` in `all-packages.nix`, the defs are also formatted as a paragraph without extra newlines. So either remove the spaces or also remove the newlines.

Reply:
The code I was replacing had this alignment, so I thought it'd be neighborly and preserve it. I've taken it out now as it didn't really help the aesthetics very much to begin with.